erpnext/stock/report/item_prices/item_prices.py

```diff
@@ -66,7 +67,6 @@ def get_printable_data(columns, data, filters):
 def get_data(filters):
 	conditions = get_item_conditions(filters, for_item_dt=False)
 	item_conditions = get_item_conditions(filters, for_item_dt=True)
-	show_amounts_role = frappe.db.get_single_value("Stock Settings", "restrict_amounts_in_report_to_role")
```

Comment:
Would changing the position of this variable cause any issue?

Reply:
Nope report working fine